### ISSUE TITLE
Update "valuable skills" section of website

### ISSUE THREAD

**Issue by antoinepetty** [srobo/website#417]
Under "VALUABLE SKILLS FOR YOUNG PEOPLE" on the homepage and /compete
"Student Robotics primes participants for good careers in engineering. Participants assemble a team, build a robot, and compete. All the while gaining valuable technical and soft skills to help their future careers."

We could reword this to be "STEM careers" or something more descriptive

**Comment by sedders123** [comment]
I don't think this sentence is too bad as it currently is?

> Student Robotics primes participants for good careers in engineering. Working to assemble a team, build a robot, and then compete, participants gain invaluable technical and soft skills to help their future careers.

That's a potential tweak we could make but I'm not sure its a clear improvement. Were you thinking of something small like this or something more radical @antoinepetty?

**Comment by antoinepetty** [comment]
I can't remember the motivation for this anymore. I think perhaps I felt "engineering" wasn't as open as "STEM"?

Either way, I like your wording. It flows better :)
